Sources/MapboxMaps/Style/Style.swift

```diff
@@ -281,13 +281,12 @@ public final class Style: StyleProtocol {
     /// Get or set the map style's transition options.
     ///
     /// By default, the style parser will attempt to read the style default
-    /// transition options, if any, falling back to an immediate transition
-    /// otherwise.
+    /// transition, if any, falling back to an immediate transition otherwise.
     ///
-    /// The style transition is re-evaluated when a new style is loaded.
-    ///
-    /// - Attention:
-    ///     Overridden transition options are reset once a new style has been loaded.
+    /// Overridden transitions are reset once a new style has been loaded.
+    /// To customize the transition used when switching styles, set this
+    /// property during `MapEvents.EventKind.styleDataLoaded`.
```

Comment:
```suggestion
    /// property after `MapEvents.EventKind.styleDataLoaded`.
```

Reply:
And should we also clarify to set it before style is fully loaded(sprite and sources are loaded)